tools/fetch_fasta_from_ncbi/retrieve_fasta_from_NCBI.py

```diff
@@ -271,7 +283,10 @@ def get_sequences(self):
                     while not mfasta:
                         self.logger.info("retrieving batch %d" % ((start / batch_size) + 1))
                         mfasta = self.efetch(self.dbname, self.query_key, self.webenv)
-                    out.write(mfasta + '\n')
+                    if ("counter" not in mfasta):
+                        out.write(mfasta + '\n')
+                    else:
+                        self.logger.info("Server Trials numbered more than 500. Skipping this batch")
```

Comment:
It would be interesting to keep this batch in raw format, to have an idea about what has been skipped (for debugging). If some NCBI entries are invalid, better to blacklist them